RecoEcal/EgammaClusterProducers/python/particleFlowSuperClusterECALOnly_cfi.py

```diff
@@ -0,0 +1,21 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoEcal.EgammaClusterProducers.particleFlowSuperClusterECALMustache_cfi import particleFlowSuperClusterECALMustache as _particleFlowSuperClusterECALMustache
+
+# "Mustache" clustering
+particleFlowSuperClusterECALOnly = _particleFlowSuperClusterECALMustache.clone(
+    # ECAL-only (no primary vertices) regression setup
+    regressionConfig = _particleFlowSuperClusterECALMustache.regressionConfig.clone(
```

Comment:
a more common/cleaner syntax here is like the following
```python
regressionConfig = dict(
    isHLT = True
...
```

Reply:
would the `dict` get merged with the existing parameters, or overwrite the full `PSet` ?

Reply:
`dict` overrides only the parameters that are given as arguments (similar to `clone()` itself).

Reply:
Thanks.

Completely unrelated: could I suggest allowing to build an `InputTag` from a `tuple`, i.e. on top of
```python
parameter = cms.InputTag("label", "instance")
parameter = "label:instance"
```
also
```python
parameter = ("label", "instance")
```
?

Reply:
> Completely unrelated: could I suggest allowing to build an `InputTag` from a `tuple`

Could you open an issue about it?